lang/cs/Org.Apache.REEF.Network/Group/Driver/Impl/MpiDriver.cs

```diff
@@ -201,6 +201,8 @@ public IConfiguration GetServiceConfiguration()
                 .BindNamedParameter<NamingConfigurationOptions.NameServerPort, int>(
                     GenericType<NamingConfigurationOptions.NameServerPort>.Class, 
                     _nameServerPort.ToString(CultureInfo.InvariantCulture))
+                .BindImplementation(GenericType<INameClient>.Class,
```

Comment:
This seems to be the only change necessary to fix REEF-202, right? What do the other changes add?